packages/main/src/themes/TimePicker.css

```diff
@@ -0,0 +1,18 @@
+:host(:not([hidden])) {
```

Comment:
We have introduced sap_belize_hcw theme before 2/3 weeks. You have to integrate with that as well, because currently the sliders are not styled, if you open the test page with ?sap-ui-theme=sap_belize_hcw